### PR TITLE
updates

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
## PR Overview
**This PR will address the following Issue/Feature:**

https://github.com/fivetran/dbt_klaviyo/issues/28

**This PR will result in the following new package version:**

v0.7.0

Removing deprecated and adding new fields. This was mostly done in the source package since most fields impacted were in `select *` statements downstream, so those models did not have to be edited, only some of the docs.

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:**

Following these [release notes](https://fivetran.com/docs/applications/klaviyo/changelog) for changes made to the connector.

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt compile
- [x] dbt run –full-refresh
- [x] dbt run
- [x] dbt test
- [ ] dbt run –vars (if applicable)

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked and tagged
- [x] You are assigned to the corresponding issue and this PR
- [x] BuildKite integration tests are passing

### Detailed Validation
Please acknowledge that the following validation checks have been performed prior to marking this PR as "ready for review":
- [x] You have validated these changes and assure this PR will address the respective Issue/Feature.
- [ ] You are reasonably confident these changes will not impact any other components of this package or any dependent packages.
- [ ] You have provided details below around the validation steps performed to gain confidence in these changes.

- dbt docs show updated ymls
- downstream models succeed 

### Standard Updates
Please acknowledge that your PR contains the following standard updates:
- Package versioning has been appropriately indexed in the following locations:
    - [x] indexed within dbt_project.yml
    - [x] indexed within integration_tests/dbt_project.yml
- [x] CHANGELOG has individual entries for each respective change in this PR
    
- [ ] README updates have been applied (if applicable)
    
- [ ] DECISIONLOG updates have been updated (if applicable)
- [ ] Appropriate yml documentation has been added (if applicable)

### dbt Docs
Please acknowledge that after the above were all completed the below were applied to your branch:
- [ ] docs were regenerated (unless this PR does not include any code or yml updates)

### If you had to summarize this PR in an emoji, which would it be?
 
:dancer:

### CODE REVIEW

**Comment by fivetran-reneeli** [comment]
I went through the models in this package and found that most of the time we're doing a select * from the source. This means that none of the deprecated columns are individually called out in the models. In addition, the new fields to be brought in will be brought downstream automatically. For models that have individual columns stated, I took an opinionated stance and concluded the new fields aren't significant enough to also be included there. 

Hence why the models themselves are unchanged!

Will wait on regen docs til after PR approved

**Review by fivetran-jamie — COMMENTED** [review]
looks pretty good -- just gotta fill in the changelog and make any changes pending source package review updates

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# dbt_klaviyo v0.7.0
+
+## 🚨 Breaking Changes 🚨:
+
```
[the remark this answers is not in the document]
Comment (fivetran-reneeli) [reply CHANGELOG.md:7]: updated

File: models/klaviyo.yml [hunk 2]
```diff
@@ -249,6 +247,10 @@ models:
         description: Timestamp of the first ever interaction between this flow and a person. 
       - name: last_event_at
         description: Timestamp of the most recent interaction between this flow and a person.
+      - name: is_archived
```
[the remark this answers is not in the document]
Comment (fivetran-reneeli) [reply models/klaviyo.yml:252]: updated

**Review by fivetran-jamie — COMMENTED** [review]
this PR looks pretty good, but i did run into an "ambiguous column" error in `klaviyo__persons` when running it. this popped up because we already have a `last_event_at` field there, calculated [here](https://github.com/fivetran/dbt_klaviyo/blob/main/models/intermediate/int_klaviyo__person_metrics.sql#L17) and now we're renaming the new `last_event_date` field as `last_event_at` [as well](https://github.com/fivetran/dbt_klaviyo_source/pull/18/files#diff-7ed2fb364b57f3f22ece830d2b18a716f4efb317f10a1b7c2b57e073a270fe76)

so we either need to rename the new field as something else (maybe leave it as `last_event_date`) or consider removing one of the `last_event_at` fields (either the new one or the old one we calculate)

after we resolve that in the source package and regen docs, i'll approve!

**Comment by fivetran-reneeli** [comment]
Thanks @fivetran-jamie ! Updated

**Review by fivetran-jamie — APPROVED** [review]
lgtm!

File: packages.yml [hunk 3]
```diff
@@ -1,3 +1,7 @@
 packages:
-- package: fivetran/klaviyo_source
-  version: [">=0.6.0", "<0.7.0"]
+# - package: fivetran/klaviyo_source
+#   version: [">=0.7.0", "<0.8.0"]
+
+  - git: https://github.com/fivetran/dbt_klaviyo_source.git
```
[the remark this answers is not in the document]
Comment (fivetran-joemarkiewicz) [reply packages.yml:5]: Bump

**Pull request merged**